README.md

```diff
@@ -1,7 +1,36 @@
-# falcon-helm
+# CrowdStrike Falcon Helm Chart and Helm Operator
+
+[Falcon](https://www.crowdstrike.com/) is the [CrowdStrike](https://www.crowdstrike.com/)
+platform purpose-built to stop breaches via a unified set of cloud-delivered
+technologies that prevent all types of attacks — including malware and much
+more.
+
+The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
+and manage the Falcon sensor on your Kubernetes cluster of choice.
+[Helm Charts](https://helm.sh/docs/topics/charts/Helm) and Helm-based Kubernetes
+Operators are packaging methodologies for Kubernetes. Which methodology you use
+is dependent on what the Kubernetes vendor and implementation supports.
+
+# Deployment
+
+There are a couple of things to understand before deploying:
+1. A Falcon sensor image must exist in your registry before deploying. See
+   https://github.com/CrowdStrike/Dockerfiles as an example of how to build a
+   Falcon sensor for your registry.
+1. The Falcon Linux (not the Falcon Container) should be used in the image to
+   deploy.
+1. Only deploying to Kubernetes Master and Worker nodes are supported at this
+   time.
+1. You need to have cluster admin privileges and be able to deploy daemonsets to
+   nodes
+1. The Helm Operator is under development and should not be used at this time.
```

Comment:
```suggestion
To ensure a successful deployment, you will want to ensure that:

1. You have access to a containerized falcon sensor image. This is most likely through a private image registry on your network or cloud provider. See https://github.com/CrowdStrike/Dockerfiles as an example of how to build a Falcon sensor for your registry.
  
1. You must have sufficient permissions to deploy Kubernetes DaemonSets. This is often received through cluster admin privileges.
   
1. CrowdStrike's Helm Operator is a project, not a product, and released to the community as a way to automate sensor deployment to kubernetes clusters. The upstream repository for this project is https://github.com/CrowdStrike/falcon-helm.
```

Reply:
* What was meant by "the falcon Linux (not the falcon container) should be used"? Was not immediately clear. 

* In regards to deploying to kube master and worker nodes, are there any other types of nodes?

* Agree the "under development" should be in big red flashing letters. Good call! Should this be a top-level paragraph vs a bullet point users may overlook?

Reply:
@shawndwells made the updates. Also, made it clear that the "Falcon Linux Sensor RPM (not the Falcon Container) should be used in the container image to deploy to Kubernetes nodes."